### PR TITLE
soc: update SoC status files

### PR DESCRIPTION
**Pull request by yijiyang**
@lumag Please help to review, thanks.

### CODE REVIEW

File: _soc/sa8775p.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ status-connectivity-ethernet: 6.6
 status-connectivity-ipa: N
 status-connectivity-wlan: N
 status-cpu-cpufreq: 6.4
+status-cpu-cpuidle: 6.10
```
Comment (lumag) [inline _soc/sa8775p.md:19]: Wasn't it supported by https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?h=v6.4&id=5d793ff40692c9b3f88df1820a79ac8ae7d9c80b ? The commit went into 6.4, if my understanding is correct.

**Review by lumag — CHANGES_REQUESTED** [review]
- Missing commit message
- The subject should provide an indea of what's going on. This one doesn't.

File: _soc/sa8775p.md (hunk 1 above)
Comment (lumag) [reply _soc/sa8775p.md:19]: Hmm, I confused cpufreq and cpuidle statuses. Then this is wrong anyway, CPUidle isn't going to be supported in 6.10 unless I missed anything.

Comment (yijiyang) [reply _soc/sa8775p.md:19]: > Hmm, I confused cpufreq and cpuidle statuses. Then this is wrong anyway, CPUidle isn't going to be supported in 6.10 unless I missed anything.

You're right. There seems to be an issue with my test routine. Indeed, this function is not fully supported.
I will close this PR.

**Pull request closed without merging**